Twispay WooCommerce Plugin/twispay/helpers/Twispay_TW_Helper_Response.php

```diff
@@ -63,8 +59,25 @@ public static function twispay_tw_decrypt_message($tw_encryptedMessage, $tw_secr
                 return FALSE;
             }
 
-            /* JSON decode the decrypted data. */
-            return json_decode($decryptedResponse, /*assoc*/TRUE, /*depth*/4);
+            /** JSON decode the decrypted data. */
+            $decodedResponse = json_decode($decryptedResponse, /*assoc*/TRUE, /*depth*/4);
+
+            /** Check if the decryption was successful. */
+              if (NULL === $decodedResponse) {
```

Comment:
in case of json_decode error it will be nice to log the json_last_error for debugging purposes.

Reply:
Solved